db/migrate/20210928144651_add_image_to_product.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageToProduct < ActiveRecord::Migration[6.0]
+  def change
+    add_column :products, :image, :string
+  end
+end
```

Comment:
Look up how to create migrations with all attributes (price, name, ...) in one migration rater than having separate migration per attribute. It is ok now but will be pain later on

Reply:
Noted, we will issue this next time we do it.

We will do this next time:

`rails generate migration AddDetailsToProjects price:decimal description:text image:string`